content/en/news/support/advance-notice-opencensus-deprecation/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Advance notice of Istio deprecating OpenCensus
+subtitle: Support Announcement
+description: Advance notice of Istio deprecating OpenCensus.
+publishdate: 2023-08-17
+---
+
+OpenCensus has [sunset](https://opentelemetry.io/blog/2023/sunsetting-opencensus) on July 31st, 2023.
+
+The OpenCensus sunset will impact Istio's usage of OpenCensus in tracing, logging, and metrics. Istio
+plans to use OpenTelemetry exporters with the OpenTelemetry collector to provide matching telemetry capabilities.
+
+This is an advance notice of Istio deprecating OpenCensus, which is tentatively scheduled to Oct. 10, 2024.
```

Comment:
Are we really going to keep it for 1 year? Seems longer than I expected.

Reply:
The deprecation policy of cloud vendors often mandates at least 12-month advance notice.

Reply:
Ok but that doesn't really have barring in what Istio does.